src/js/view/view.js

```diff
@@ -716,6 +724,17 @@ define([
                 default:
                     break;
             }
```

Comment:
@hghazzi I didn't think it was worth it (to keep it short requires fall-throughs and inner if statements. Can you provide an example? Maybe just the first one is two if statements.